src/Read_Forcing.f90

```diff
@@ -13,55 +13,56 @@ SUBROUTINE Read_Forcing
 no_flow=0
 no_heat=0
 do nr=1,nreach
-  do nc=1,no_cells(nr)-1 
-    no_flow=no_flow+1
-    no_heat=no_heat+1
+  do nc=1,no_cells(nr)-1 
+    no_flow=no_flow+1
+    no_heat=no_heat+1
 !
-    nrec_flow=flow_cells*(ndays-1)+no_flow
-    nrec_heat=heat_cells*(ndays-1)+no_heat
+    nrec_flow=flow_cells*(ndays-1)+no_flow
+    nrec_heat=heat_cells*(ndays-1)+no_heat
 !
-    read(35,'(2i5,2f10.1,2f6.1,f7.1,f6.2)' &
-           ,rec=nrec_flow) nnd,ncell &
-           ,Q_out(no_heat),Q_dmmy,Q_diff(no_heat) &  
-           ,depth(no_heat),width(no_heat),u(no_heat)
+    read(35,'(2i5,2f10.1,2f6.1,f7.1,f6.2)' &
+           ,rec=nrec_flow) nnd,ncell &
+           ,Q_in(no_heat),Q_out(no_heat),Q_diff(no_heat) &  
+           ,depth(no_heat),width(no_heat),u(no_heat)
 !
-    if(u(no_heat).lt.0.01) u(no_heat)=0.01
-    if(ncell.ne.no_heat) write(*,*) 'Flow file error',ncell,no_heat 
+    if(u(no_heat).lt.0.01) u(no_heat)=0.01
+    if(ncell.ne.no_heat) write(*,*) 'Flow file error',ncell,no_heat 
 !
-    read(36,'(i5,2f6.1,2f7.4,f6.3,f7.1,f5.1)' &
-           ,rec=nrec_heat) ncell &
-           ,dbt(no_heat),ea(no_heat) &
-           ,Q_ns(no_heat),Q_na(no_heat),rho &
+    read(36,'(i5,2f6.1,2f7.4,f6.3,f7.1,f5.1)' &
+           ,rec=nrec_heat) ncell &
+           ,dbt(no_heat),ea(no_heat) &
+           ,Q_ns(no_heat),Q_na(no_heat),rho &
            ,press(no_heat),wind(no_heat)
-!          
-  if(ncell.ne.no_heat) write(*,*) 'Heat file error',ncell,no_heat 
-!
-!  Added variable ndelta (UW_JRY_2011/03/15
-
-!
-    delta_n=ndelta(ncell)
-! 
-    Q_avg=0.5*(Q_in(no_heat)+Q_out(no_heat))
-    dt(no_heat)=dx(no_heat)/u(no_heat)
+!          
+!  if(ncell.ne.no_heat) write(*,*) 'Heat file error',ncell,no_heat 
+!
+!  Added variable ndelta (UW_JRY_2011/03/15
+!
+    delta_n=ndelta(ncell)
+! 
+    Q_avg=0.5*(Q_in(no_heat)+Q_out(no_heat))
+    Q_diff(no_heat)=Q_diff(no_heat)/delta_n
```

Comment:
Introducing Q_diff makes me think of a side comment - this isn't a comment on the code as much as a suggestion.  I wonder if we A) define what each variable means at the top of each script or B) include in the "ReadMe" the definition of the variables.  I currently have an informal document that does this.  To be fair - I need to improve this in my code (clearly defining variables in comments).